test/blackbox-tests/test-cases/github2206.t/run.t

```diff
@@ -1,4 +1,7 @@
 copy_files would break the generation of the preprocessing flags
+
+  $ export DUNE_CONFIG__SKIP_LINE_BREAK=enabled
```

Comment:
I didn't follow the original feature, but maybe it would be simpler to make that just an `INSIDE_DUNE` thing?

Reply:
In the long run I would like to enable this by default (or at least configure the terminal width to be be margin with by default), I think Dune is much better with this option even outside of the tests.

Reply:
(sorry for the off-topic discussion)

I think it's a good idea to adapt to the size of the terminal, but this will require tinkering with SIGWINCH etc - I think that notty already handles that so it's possible that the terminal UI already works that way.

Exposing this to the user might be confusing so I'm wondering if there's a heuristic we can use. When will the user want to set this variable, and when won't they?

Reply:
My thinking is that the user (a fancy way of saying "me") basically never wants wrapping to happen. Even if we wrap at the width of the terminal, we're wrapping at a place where the terminal would've wrapped anyway and we introduce a hard-newline. So if the user resizes the terminal afterwards, it can't rewrap the text anyway, there will always be the hard-newline. If OTOH we just leave it up to the terminal it works.

The only place where wrapping makes some sense IMHO is when we want to print some hierarchical structure with indentation and long lines and then the terminal softwrapping would break it. Even in these cases at least I personally prefer to disable line-wrapping in the terminal and have side-scrolling.

However, this might be a better setup for a dedicated RFC to explain the idea to a bigger audience and solicit more perspectives.

Reply:
That sounds useful. Another reason to customize this is that some people prefer a different margin anyway. From what I remember, 120 chars was asked before.